stable/mongodb-replicaset/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mongodb-replicaset
 home: https://github.com/mongodb/mongo
-version: 3.11.6
+version: 3.11.7
```

Comment:
`3.12.0` for new features

Reply:
changed to 3.13.0 as 3.12.0 is already taken